seed.js

```diff
@@ -1,15 +1,15 @@
 // This file allows us to seed our application with data
 // simply run: `node seed.js` from the root of this project folder.
 
-// var db = require('./models');
+var db = require('./models');
 
-// var new_campsite = {description: "Sharp rocks. Middle of nowhere."}
+var new_hobby = {description: "Sharp rocks. Middle of nowhere."}
 
-// db.Campsite.create(new_campsite, function(err, campsite){
-//   if (err){
-//     return console.log("Error:", err);
-//   }
+db.Hobby.create(new_hobby, function(err, hobby){
```

Comment:
This seed data is based on the Hobby that you have completely commented out in your code--it's clear you didn't put in the effort you could have here.